tiny/src/conn.rs

```diff
@@ -261,6 +262,7 @@ fn handle_irc_msg(ui: &dyn UI, client: &Client, msg: wire::Msg) {
                     &MsgTarget::Chan { serv, chan: &chan },
                 );
             }
+            ui.set_tab_style(TabStyle::JoinOrPart, &MsgTarget::Chan { serv, chan: &chan })
```

Comment:
I think this is not buggy, but I think it would be better to move this to the `if` above, for similar reasons.